dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -155,9 +155,8 @@ def facilitator_pre_workshop(user, workshop)
   def facilitator_post_workshop(user, workshop)
     @user = user
     @workshop = workshop
-    survey_params = "survey_data[workshop_course]=#{workshop.course}&survey_data[workshop_subject]=#{workshop.subject}"\
-                    "&survey_data[workshop_id]=#{workshop.id}"
-    @survey_url = CDO.studio_url "form/facilitator_post_survey?#{survey_params}", CDO.default_scheme
+    survey_params = "workshop_id=#{workshop.id}"
```

Comment:
question for reviewers: any hidden reason survey params should include anything besides workshop id?

Reply:
I don't see a reason to need the extra params. It looks like this logic may have been doing what `survey_name` now does? Not sure

Reply:
thanks for looking. after further investigation, I believe these should be safe to remove. 
* these params are needed to display the form title: https://github.com/code-dot-org/code-dot-org/blob/2e96767b96725eda04d4d3687dab3fe9049e50e1/dashboard/config/foorm/forms/surveys/pd/csd_csp_facilitator_post_survey.1.json#L10
* the old codepath depended on these params to be passed in explicitly: https://github.com/code-dot-org/code-dot-org/blob/b95ff13801571ab9a9a650d9082bd025d0b48da6/dashboard/app/controllers/foorm/simple_survey_forms_controller.rb#L85
* the new codepath does the work to pull these out of the workshop: https://github.com/code-dot-org/code-dot-org/blob/305ab1e3ea5a3c905dc2e4f6b0847d2e020f8304/dashboard/app/controllers/pd/workshop_daily_survey_controller.rb#L330-L331

Reply:
I also verified that the title is still displaying course and subject locally:
![Screenshot 2023-03-09 at 12 21 31 PM](https://user-images.githubusercontent.com/8001765/224146589-68fa5edc-fc8b-4b73-8d68-f22cff5b2e7d.png)

Reply:
Wow, thorough investigation! This looks good 👍